Sources/TuistKit/Models/ProjectStructure.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+struct ProjectStructure {
```

Comment:
Considering that this type only contains a string attribute, what about using a String instead? Where you thinking about this struct getting more attributes in the future?

Reply:
possibly - I had a prototype where I customized all the group names but felt that was excessive. There is a small syntactic sugar bonus we get with this with the static initializers `.default` and in my other diff `.manifestTargetStructure` with some type-safety :)

Reply:
I feel it's too much. The `Project` can be a constant in the class that generates the groups. Just trying not to overcomplicate things.

Reply:
If I understood correctly - just use a string directly:

```swift
// GeneratorModelLoader.swift

func from(manifest: ... ) {
     return Target(name: name,
                      platform: platform,
                      product: product,
                      // ...
                      filesGroup: "Project",  // or `sourcesGroup` ?
                      dependencies: dependencies)
}
```

How about an enum instead to make it even clearer:

e.g.

```swift
enum ProjectGroup {
    case root // or `main`, this can be added once I figure out a cleaner way to do the sorting
    case group(name: String)
}
```


```swift
// GeneratorModelLoader.swift

func from(manifest: ... ) {
     return Target(name: name,
                      platform: platform,
                      product: product,
                      // ...
                      filesGroup: .group(name:"Project"), // .group(name: "Manifest"), .root
                      dependencies: dependencies)
}
```

Reply:
I lean more towards the enum implementation.